effects/bouncingballeffect.h

```diff
@@ -39,11 +39,6 @@ class BouncingBallEffect : public LEDEffectBase
     {
     }
 
-    inline static string EffectTypeName() 
-    { 
-        return typeid(BouncingBallEffect).name();
-    }
-
```

Comment:
The effect type name is now added in the `to_json()` function for `ILEDEffect`, for all effects. I figured this made more sense, because it's the corresponding `from_json()` function that uses it.